### ISSUE TITLE
photon-setup no longer accepts highly available lw setups

### ISSUE THREAD

**Issue by pompomJuice** [vmware/photon-controller#105]
photon-setup no longer accepts DNS IPs consisting of more than one IP.

The new error given on reference config pc-config.yaml
`2017-04-18 11:23:27 ERROR Error: network-config for controller 'lightwave-1' has invalid DNS IP address '10.118.102.12,10.118.102.13'`

I was wondering if this is intended?

**Comment by AlainRoy** [comment]
You're correct--this is a regression due to a well-intentioned code change to do more upfront validation of the installer's configuration.

I've filed this internally as bug #1853506 and we'll fix it in the 1.2.1 release. While I don't have a date for that release, it's "weeks" not "months".

**Comment by bhundven** [comment]
...or maybe "months"?

**Comment by It4lik** [comment]
The 1.2.1 is actually out. If you go to the main release page, only 1.2.0 shows up but you can easily find 1.2.1. 

That being said, this solution looks like it's definitely dead yep

**Comment by mwest44** [comment]
We are going through a process of changing the go to market strategy for this product.  Those changes may impact how you consume the product, however Development is ongoing.  Look for an update in the next couple of weeks.

**Comment by It4lik** [comment]
Okay. But will it still be based on top of Xenon ? Will you keep on promoting PivotalCF+Photon Controller ?  When you talk about "changing the market strategy", will that impact the real goal of Photon Controller (like enable powerful IaaS with full control for IT teams, and nice user/permission/resources management) ? Or maybe it will be deeply technically impacted ? 

Most importantly, will this become fully free/open-source ?
Will you continue to develop over this project or start from scratch again ? What form of support can we expect ? So many questions in my mind, I'm looking forward in the next weeks to see where this is going. 

But please, give us reasons to trust this project. I think it has a huge potential. Why does this project lacks so much of communication where Harbor or Admiral is full of commits/issues answers everyday ?

**Comment by mwest44** [comment]
All good questions.  I am going to hold on giving partial responses and wait for all of the details to be committed.

**Comment by It4lik** [comment]
Mmh okay, no problem. I'll stay tuned. Or maybe "we" ? @pompomJuice  ;)

**Comment by pompomJuice** [comment]
This stuff is the future! Lets us at it!
